enterprise/coderd/licenses.go

```diff
@@ -253,7 +254,7 @@ func decodeClaims(l database.License) (jwt.MapClaims, error) {
 	if len(parts) != 3 {
 		return nil, xerrors.Errorf("Unable to parse license %d as JWT", l.ID)
 	}
-	cb, err := base64.URLEncoding.DecodeString(parts[1])
+	cb, err := base64.RawURLEncoding.DecodeString(parts[1])
```

Comment:
For those who had to look this up, there are four kinds of base64 encoding [1]  [2]:
- `base64.URLEncoding`: filename and URL-safe encoding, with padding
- `base64.RawURLEncoding`: filename and URL-safe encoding, without padding
- `base64.StdEncoding`: not safe for use in filenames and URLs, with padding
- `base64.RawStdEncoding`: not safe for use in filenames and URLs, without padding

[1] https://pkg.go.dev/encoding/base64#pkg-variables
[2] https://www.rfc-editor.org/rfc/rfc4648.html#section-4